src/main/java/com/bsu/TweetCollection.java

```diff
@@ -0,0 +1,39 @@
+package com.bsu;
+
+import java.util.ArrayList;
+
+public class TweetCollection {
+    
+    private ArrayList<Tweet> tweets;
```

Comment:
Prefer abstract interfaces on declaration level ( List<Tweet> in that situation )